skycatalogs/skyCatalogs.py

```diff
@@ -334,6 +334,9 @@ def __init__(self, config, mp=False, skycatalog_root=None, verbose=False,
                                                                     'sso_sed.db')
 
             self._sso_sed_factory = SsoSedFactory(self._sso_sed_path)
+            if not self._sso_sed_factory:
+                self._logger.warning('SSO appear in the list of available object types but supporting files do not exist')
+                self._logger.warning('SSOs will not be present in object lists')
```

Comment:
I think that a warning that explicitly says that "SSOs will not be simulated" would be more helpful to users.